pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -21,9 +21,9 @@ stdenvNoCC.mkDerivation rec {
         sha256 = "1bf7i54iqlf7x7fn8kqwmi06g30sxr6nq3ac0r871i6g0p3y47sf";
       });
 
-  buildInputs = if stdenvNoCC.isDarwin then [ undmg ] else [ rpmextract ];
+  buildInputs = if stdenv.isDarwin then [ undmg tbb fixDarwinDylibNames ] else [ rpmextract ];
```

Comment:
`fixDarwinDylibNames`, `rpmextract` and `undmg` should probably go to `nativeBuildInputs`. You could also add `darwin.cctools` there to provide you install_name_tool.

Reply:
OK, done.

Reply:
So no need to change `stdenvNoCC` to `stdenv` now?

Reply:
You're right. I've changed it back to `stdenvNoCC`.